assets/css/main.scss

```diff
@@ -1,8 +1,133 @@
 ---
 ---
 
```

Comment:
Possibly add note to not edit this file and rather include your custom CSS in `custom.scss`

Reply:
I'm wondering if there is a way to keep this file inside of the `_sass` directory so when the template is used the only scss file is `site.scss` ?

Reply:
I think we would need to keep this file here because none of the files in `_sass` are actually generated by Jekyll when the site in generated. Keeping this file here let's us access it using the path `assets/css/main.css` which is what is referenced by the `styles.html` include.

Reply:
I like @hursey013's idea to add a comment up to the top of the file that this shouldn't need to be edited